aws/data_source_aws_dynamodb_table.go

```diff
@@ -168,6 +168,12 @@ func dataSourceAwsDynamoDbTable() *schema.Resource {
 				Computed: true,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"kms_key_arn": {
+							Type:         schema.TypeString,
+							Optional:     true,
+							Computed:     true,
+							ValidateFunc: validateArn,
```

Comment:
Since this attribute is not configurable in the data source, `Optional` nor validation should be specified in the schema. 👍 

```suggestion
							Type:         schema.TypeString,
							Computed:     true,
```

Reply:
Thanks for this feedback, @bflad. I think I understand but to make sure: does that mean that the `Optional: true` about a dozen lines down in `server_side_encryption` is also unnecessary?